storage/flux/table.gen.go.tmpl

```diff
@@ -5,8 +5,8 @@ import (
 	"math"
```

Comment:
Changes to this file and the `table.gen.go` above are from b1aa943e1b1548a279febf31bdbe0ebde02df7b1